radio/src/targets/nv14/board.cpp

```diff
@@ -176,6 +176,11 @@ void boardInit()
   lcdInit();
   usbInit();
   hapticInit();
+
+ #if defined(RTCLOCK) && !defined(COPROCESSOR)
```

Comment:
I think the `COPROCESSOR` def relates to the SKY9X board, which means it is useless here. Can you please remove it?